packages/block-library/src/blocks/slideshow/inspector-controls.js

```diff
@@ -105,27 +122,94 @@ const SlideshowInspectorControls = function( props ) {
 				</ControlsSection>
 			}
 
-			{
-				'gallery' === slideshowType &&
-				<Fragment>
-					<LayoutControls { ...props } />
-					<ControlsSection label={ __( 'Layout' ) }>
-						<ControlsTab label={ __( 'Settings' ) }>
-							<RadioControl
-								key={ 'slideshow-minimum-height-controls' }
-								label={ __( 'Minimum Height', '__plugin_txtd' ) }
-								selected={ minHeight }
-								onChange={ ( nextMinHeight ) => {
-									setAttributes( { minHeight: parseInt( nextMinHeight, 10 ) } );
-								} }
-								options={ minHeightOptions }
-							/>
-						</ControlsTab>
-					</ControlsSection>
-				</Fragment>
-			}
+
+      <Fragment>
+        <LayoutControls { ...props } />
+        <ControlsSection label={ __( 'Layout' ) }>
+          <ControlsTab label={ __( 'Settings' ) }>
+            <RadioControl
+              key={ 'slideshow-minimum-height-controls' }
+              label={ __( 'Minimum Height', '__plugin_txtd' ) }
+              selected={ minHeight }
+              onChange={ ( nextMinHeight ) => {
+                setAttributes( { minHeight: parseInt( nextMinHeight, 10 ) } );
+              } }
+              options={ minHeightOptions }
+            />
+          </ControlsTab>
+        </ControlsSection>
+      </Fragment>
+
+      {
+        'gallery' !== source &&
+        <ControlsSection label={ __( 'Display' ) } group={ __( 'Block Modules' ) }>
+          <ControlsTab label={ __( 'Settings' ) }>
+            <ControlsGroup title={ __( 'Set up elements for this block', '__plugin_txtd' ) }>
+              <ToggleGroup
+                onChange={ setAttributes }
+                toggles={ toggles.filter( toggle => {
+                  return toggle.attribute !== 'showMedia' &&
+                         toggle.attribute !== 'showSubtitle' &&
+                         toggle.attribute !== 'showCollectionSubtitle' &&
+                         toggle.attribute !== 'showCollectionTitle';
+                } ).map( toggle => {
+                  return {
+                    ...toggle,
+                    value: attributes[ toggle.attribute ]
+                  }
+                } ) }
+              />
+            </ControlsGroup>
+            <MetaSource { ...props } />
+          </ControlsTab>
+        </ControlsSection>
+      }
 		</Fragment>
 	);
 };
 
+const MetaSource = ( props ) => {
```

Comment:
MetaSource and ToggleGroup should be moved inside a hook in the block-editor package. the ToggleGroup component should receive a function through props in order to filter the displayed toggles. At the same time, attributes needed for these components should be moved to that hook as well.